tutorials/ui/bbcode_in_richtextlabel.rst

```diff
@@ -931,10 +932,35 @@ Ghost
         char_fx.color.a = alpha
         return true
 
+ .. code-tab:: csharp
+ 
+    using Godot;
+
+    [Tool]
+    public class RichTextGhost : RichTextEffect
+    {
+        // Syntax: [ghost freq=5.0 span=10.0][/ghost]
+
+        // Define the tag name.
+        private const string Bbcode = "ghost";
```

Comment:
Sorry about this, as explained in https://github.com/godotengine/godot-docs/pull/9420#discussion_r1616332119 this needs to be named exactly `bbcode`.

```suggestion
        private string bbcode = "ghost";
```